packages/babel-cli/test/fixtures/babel-node/--no-babelrc/.babelrc

```diff
@@ -0,0 +1,3 @@
+{
```

Comment:
I'm surprised this isn't inside `in-files`. Same for the plugin. I think if you move them in there you'd be able to use `./logPlugin.js`?